libs/iblprefilter/src/materials/equirectToCube.mat

```diff
@@ -0,0 +1,78 @@
+material {
+    name : equirectToCube,
+    parameters : [
+        {
+            type : sampler2d,
+            name : equirect,
+            precision: medium
+        },
+        {
+            type : float,
+            name : side,
+            precision: medium
+        }
+    ],
+    outputs : [
+        {
+            name : outx,
+            target : color,
+            type : float3
+        },
+        {
+            name : outy,
+            target : color,
+            type : float3
+        },
+        {
+            name : outz,
+            target : color,
+            type : float3
+        }
+    ],
+    variables : [
+        vertex
+    ],
+    domain : postprocess,
+    depthWrite : false,
+    depthCulling : false
+}
+
+vertex {
+    void postProcessVertex(inout PostProcessVertexInputs postProcess) {
+        postProcess.vertex.xy = postProcess.normalizedUV;
+    }
+}
+
+fragment {
+
+void dummy() {}
+
+highp vec2 toEquirect(const highp vec3 s) {
+    highp float xf = atan2(s.x, s.z) * (1.0 / PI);  // range [-1.0, 1.0]
+    highp float yf = asin(s.y) * (2.0 / PI);        // range [-1.0, 1.0]
+    xf = (xf + 1.0) * 0.5;                          // range [0, 1.0 [
+    yf = (1.0 - yf) * 0.5;                          // range [0, 1.0 [
```

Comment:
either a typo or a range notation that I'm not familiar with :)

Reply:
It means the 1.0 is exclusive.

Reply:
Ah yeah, correct, it's actually inclusive -- this is because this comment comes from the CPU version.